### DISCUSSION TITLE
Mounting a Volume to Media Server For playing audio files from host machine

### DISCUSSION THREAD

**Discussion by harish-chander** [fonoster/yaps#269] [Q&A]
Does any one know which Config file need to be modified for adding a docker volume mount which will mount to mediaserver  container (asterisk) for Playing Audio files using res.play("sound:xyz.wav") or res.play("sound:/path/to/file/zyc.wav") . instead of using web/restapi resource .

**Comment by psanders** [comment]
@harish-chander 

My recommendation is to create a folder on the host machine with all the audio files you want. Then, you can add the folder as a volume at the [02_sipnet.yml](https://github.com/fonoster/fonoster/blob/bb37ed610cd752a7d27c7560582ba658db975036/operator/compose/02_sipnet.yml#L91)

Note: You need to mount your files at `/var/lib/asterisk/sounds`.

**Comment by harish-chander** [comment]
Noted thanks @psanders  . I will test it out
